Header header
uint8 direction
uint8 mode
